src/remote/watcher.coffee

```diff
@@ -77,8 +79,7 @@ class RemoteWatcher
                         @whenReady callback
             else
                 @changes = @couch.client.changes
-                    filter: (doc) ->
-                        doc.docType?.toLowerCase() in ['file', 'folder']
+                    filter: @filterName
```

Comment:
yes, it's the same but why I pass by view if filter can do it ? Maybe I don't know a things ?

Exemple 3:

    db.changes({
      filter: 'mydesign/myfilter'
    });

Exemple 5:

    db.changes({
      filter: '_view',
      view: 'mydesign/myview'
    });